src/ol/source/wmts.js

```diff
@@ -444,6 +449,7 @@ ol.source.WMTS.optionsFromCapabilities = function(wmtsCap, config) {
     tileGrid: tileGrid,
     style: style,
     dimensions: dimensions,
-    wrapX: wrapX
+    wrapX: wrapX,
+    crossOrigin: crossOrigin
```

Comment:
The value should be `undefined` by default, same as the `crossOrigin` option for the WMTS layer.
The code lines 437-440 can be removed and this line should be:
```
crossOrigin: config['crossOrigin']
```